be/src/block_cache/block_cache.h

```diff
@@ -0,0 +1,47 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include "block_cache/kv_cache.h"
+#include "common/status.h"
+
+namespace starrocks {
+
+class BlockCache {
+public:
+    typedef std::string CacheKey;
+
+    ~BlockCache();
+
+    // Return a singleton block cache instance
+    static BlockCache* instance();
+
+    // Init the block cache instance
+    Status init(const CacheOptions& options);
+
+    // Set block size as the cache unit
+    Status set_block_size(size_t block_size);
+
+    // Write data to cache, the offset must be aligned by block size
+    Status write_cache(const CacheKey& cache_key, off_t offset, size_t size, const char* buffer,
```

Comment:
nit: put/get seems much general

Reply:
> nit: put/get seems much general

`put/get` seems more suitable in some kv interfaces, but our block cache interface is a top layer based on kv cache and it still contains some io parameters like `offset`, `size`, etc.